src/index.jsx

```diff
@@ -37,8 +38,14 @@ function renderComponentInElement(el) {
   if (el.dataset.type === 'resource-preview') {
     let idx = parseInt(el.dataset.resource)
     let resource = viewutils.findResourceByNameOrIndex(dp, idx)
-    ReactDOM.render(<HandsOnTable resource={resource} idx={idx} />, el);
+    if (resource.format === 'geojson') {
+      ReactDOM.render(<LeafletMap featureCollection={resource._values} idx={idx} />, el)
+    } else {
+      ReactDOM.render(<HandsOnTable resource={resource} idx={idx} />, el)
+    }
   } else if (el.dataset.type === 'data-views') {
-    ReactDOM.render(<MultiViews dataPackage={dp} />, el);
+    if (dp.views !== undefined) {
```

Comment:
I have fixed it in MultiViews.jsx - when there is no views given, it renders empty div element.